tests/unit/test_cubepart.py

```diff
@@ -225,6 +225,10 @@ def it_provides_the_transforms_dict_to_help(self, transforms, expected_value):
         """Handles defaulting of transforms arg."""
         assert CubePartition(None, transforms)._transforms_dict == expected_value
 
+    def it_knows_its_selected_categories_labels(self):
+        # --- default of {} is overridden by subclasses when appropriate ---
```

Comment:
Yeah, this makes no sense to me at first sight. We should _only_ have it in derived classes. Higher classes don't know about that case - it's too specific.

Reply:
There's generally no need to test an "interface-defining" method on a base class that raises `NotImplementedError`.